main.cpp

```diff
@@ -238,24 +213,23 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    guessTheNumber(7, 3);
-    ignoreUnused(guessTheNumber);
+    auto numberGuessed = guessTheNumber(7, 3);
+    ignoreUnused(numberGuessed);
     
     //2)
-    doSomeMath(1, 300, 3.14f);
-    ignoreUnused(doSomeMath);
+    auto mathDone = doSomeMath(1, 300, 3.14f);
+    ignoreUnused(mathDone);
     
     //3)
-    checkTheStatus(1);
-    ignoreUnused(checkTheStatus);
+    auto statusChecked = checkTheStatus(1);
+    ignoreUnused(statusChecked);
 
     //4)
-    questionExistence(28, 45);
-    ignoreUnused(questionExistence);
+    questionExistence(28, 45);
     
     //5)
-    calculateTheThing(4.56f, 3.156f, 6.3214554f);
-    ignoreUnused(calculateTheThing);
+    auto thingCalculated = calculateTheThing(4.56f, 3.156f, 6.3214554f);
+    ignoreUnused(thingCalculated);

```

Comment:
pass returned values (stored in a local variable) to a single call of `ignoreUnused`
- [ ] revise